test/test_get.py

```diff
@@ -259,6 +263,94 @@ def test_model_methods_each_called_twice_when_two_version_ids_given(
             assert mock_output.call_args_list == [call(), call()]
             assert result.exit_code == 0
 
+        @patch.object(version_history.ModelVersionHistory, "output_version_history")
+        @patch.object(version_history.ModelVersionHistory, "__init__")
+        def test_version_history_methods_each_called_once_when_one_version_id_given(
+            self,
+            mock_initialise,
+            mock_output_version_history,
+            mock_jwt,
+            mock_get,
+            mock_set_metadata,
+            mock_output,
+            mock_get_metadata,
+            mock_details,
+            processed_jwt_fixture,
+            get_models_list_fixture,
+            get_model_metadata_fixture,
+        ):
+            # SETUP
+            # setup get jwt
+            mock_jwt.return_value = processed_jwt_fixture, False
+            # setup retrieving
+            mock_get.side_effect = [
```

Comment:
this can be `mock_get.side_effect = get_models_list_fixture`, as the fixture is an iterable

Reply:
Good spot. Changed.